src/mesh/RadioLibInterface.cpp

```diff
@@ -270,9 +276,16 @@ void RadioLibInterface::onNotify(uint32_t notification)
                 if (delay_remaining > 0) {
                     // There's still some delay pending on this packet, so resume waiting for it to elapse
                     notifyLater(delay_remaining, TRANSMIT_DELAY_COMPLETED, false);
+#if !MESHTASTIC_EXCLUDE_TIPS
+                } else if (MeshTipsModule::configureRadioForPacket(this, txp)) {
+                    // We just switched radio config, so wait to ensure the new channel is available
+                    setTransmitDelay();
```

Comment:
This is probably fine for the "Tips" broadcast, but during the switching time and further delays from e.g. CSMA/CA, you'll be on the non-default frequency and hence you cannot receive from the default frequency. Have you measured how long the switching takes such that we can optimize the delay?

Furthermore, during this time packets in the queue can get reordered or canceled, so you might be switching around without actually transmitting on the other frequency. I think in this period we should avoid changing the packet in front of the queue (except for canceling it).

Reply:
> During... you'll be on the non-default frequency & cannot receive.

Yes. This is an expected tradeoff of using the feature - the radio can only do one thing at a time.

> Have you measured how long the switching takes such that we can optimize the delay?

No formal measurement. However, the delay is quite small (milliseconds), and IMO reasonable to consider part of the tradeoff. Would formal measurements be useful? Happy to spend some time benchmarking radio times if so.

> I think in this period we should avoid changing the packet in front of the queue (except for canceling it).

This is an extremely good point, and I agree - thanks for the suggestion.